### PR TITLE
fix: python context relevancy cookbook

### PR DESCRIPTION
**Pull request by Matthieu-OD**
Based on this [issue](https://github.com/explodinggradients/ragas/issues/1210#issuecomment-2303938086), in `ragas.metrics`, `context_relevancy` has been renamed to `answer_relvancy`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@
 **/dist
 **/node_modules
 
-.env
\ No newline at end of file
+.env
+.python-version
```
Comment (Matthieu-OD) [inline .gitignore:9]: I'm using `pyenv` this file should be ignored

File: python/context-relevancy-ragas/Context relevancy with Ragas.ipynb [hunk 2]
```diff
@@ -77,6 +77,7 @@
     "from dotenv import load_dotenv\n",
     "from literalai import LiteralClient\n",
     "\n",
+    "os.environ.clear()\n",
```
Comment (Matthieu-OD) [inline python/context-relevancy-ragas/Context relevancy with Ragas.ipynb:80]: ensure I'm using the correct `LITERAL_API_KEY` (the one from my .env file)

File: python/context-relevancy-ragas/Context relevancy with Ragas.ipynb [hunk 3]
```diff
@@ -309,23 +295,29 @@
...
-    "    params=[{ \"type\": context_relevancy.name, \"top_k\": 2 }]\n",
+    "    params=[{ \"type\": answer_relevancy.name, \"top_k\": 2 }]\n",
     ")\n",
     "\n",
     "# Log each experiment result.\n",
     "for index, row in results.iterrows():\n",
+    "    relevancy_score = row[answer_relevancy.name]\n",
+    "    if pd.isna(relevancy_score):\n",
+    "        relevancy_score = 0\n",
+    "\n",
```
Comment (Matthieu-OD) [inline python/context-relevancy-ragas/Context relevancy with Ragas.ipynb:316]: avoid having an error if `relevancy_score = NaN`

File: python/context-relevancy-ragas/Context relevancy with Ragas.ipynb [hunk 4]
```diff
@@ -394,23 +371,27 @@
...
-    "    params=[{ \"type\": context_relevancy.name, \"top_k\": 1 }]\n",
+    "    params=[{ \"type\": answer_relevancy.name, \"top_k\": 1 }]\n",
     ")\n",
     "\n",
     "# Log each experiment result.\n",
     "for index, row in results.iterrows():\n",
+    "    relevancy_score = row[answer_relevancy.name]\n",
+    "    if pd.isna(relevancy_score):\n",
+    "        relevancy_score = 0\n",
+    "\n",
```
Comment (Matthieu-OD) [inline python/context-relevancy-ragas/Context relevancy with Ragas.ipynb:390]: avoid having an error if `relevancy_score = NaN`

**Review by desaxce — APPROVED** [review]
lgtm

**Pull request merged**